src/components/cavatica/CavaticaFileSummary.js

```diff
@@ -51,7 +51,7 @@ const enhance = compose(
           if (!studies[study]) {
             studies[study] = [];
           }
-          studies[study].push(file.node.uuid);
+          studies[study].push(file.node.kf_id);
```

Comment:
this is `kf_id` rather than `latest_did` intentionally?

Reply:
That's what's quoted in issue report, but @rosibaj could you confirm just in case ? :) https://github.com/kids-first/kf-portal-ui/issues/430

Reply:
```latest_did``` is correct - sorry about that @hlminh2000 - i correct #434 but not #430 when i sent it over to you today

Reply:
ah thanks @rosibaj!!! will update!